security-malware-firewall.php

```diff
@@ -544,6 +544,8 @@ function spbc_activation( $network, $redirect = true ) {
 	// Redirect
 	if($redirect)
 		add_option('spbc_activation__redirect', $spbc->settings_link );
+
+	spbc_activation__maybe_get_api_key();
```

Comment:
Can rename the function as **spbc_activation__try_get_api_key**
maybe - it looks unusual

Reply:
_spbc_activation__maybe_get_api_key_ - Works for me. And I think it sounds better in this context.

Reply:
Leave "maybe"